### DISCUSSION TITLE
Can't Find old docs

### DISCUSSION THREAD

**Discussion by MarouaniALA** [primefaces/community#3269] [Show and tell]
Hi where can i found doc older than V17

**Comment by TheHolyTurtle** [comment]
I have the same issue! I am in the process of updating a legacy project from version 11 of angular all the way to version 18, and we make use of some old components from primeNG that i needed the docs to make sure they don't break. I used to click on the lts 11 version of PrimeNG from the dropdown, but the list only shows from v17-v19 now?!
